### PR TITLE
Add tz.tuya_thermostat_away_mode (fix for https://github.com/Koenkk/z…

### PR DESCRIPTION
**Pull request by hacker-cb**
…igbee2mqtt/pull/4491)

### CODE REVIEW

**Comment by Koenkk** [comment]
I expect a `away_mode` converter to be added here (https://github.com/Koenkk/zigbee-herdsman-converters/blob/e7a89526a0cc55967e84d1b315642b392db94d6d/devices.js#L1495), otherwise this won't work?

**Comment by Hacker-CB** [comment]
Sorry, seems that I forgot to create second PR.
I just created it https://github.com/Koenkk/zigbee-herdsman-converters/pull/1618

File: lib/extension/homeassistant.js [hunk 1]
```diff
@@ -867,15 +867,15 @@ const climate = (minTemp=7, maxTemp=30, temperatureStateProperty='occupied_heati
...
-        // const indexOfAway = holdModes.indexOf('away');
-        // if (indexOfAway > -1) {
-        //     holdModes.splice(indexOfAway, 1); // HA will add "Away" to modes by itself
-        //     retVal.discovery_payload.away_mode_command_topic = true;
-        //     retVal.discovery_payload.away_mode_state_topic = true;
-        //     retVal.discovery_payload.away_mode_state_template =
-        //         '{{ value_json.away_mode }}';
-        // }
-        /* istanbul ignore else */
+        const indexOfAway = holdModes.indexOf('away');
```
Comment (Koenkk) [inline lib/extension/homeassistant.js:870]: Could you add `/* istanbul ignore else */` above here so that tests pass? (somehow editing is disabled for me)

**Comment by Hacker-CB** [comment]
@Koenkk this also please merge :)

**Comment by Koenkk** [comment]
@Hacker-CB please check https://github.com/Koenkk/zigbee2mqtt/pull/4510#discussion_r500290183

File: lib/extension/homeassistant.js (hunk 1 above)
Comment (Hacker-CB) [reply lib/extension/homeassistant.js:817]: > @Hacker-CB please check [#4510 (comment)](https://github.com/Koenkk/zigbee2mqtt/pull/4510#discussion_r500290183)

I Added in https://github.com/Koenkk/zigbee2mqtt/pull/4510/commits/b7f104aaf5112a999ba6f7cde75ee59eec92dfa3

**Comment by Hacker-CB** [comment]
> @Hacker-CB please check [#4510 (comment)](https://github.com/Koenkk/zigbee2mqtt/pull/4510#discussion_r500290183)

I added in https://github.com/Koenkk/zigbee2mqtt/pull/4510/commits/b7f104aaf5112a999ba6f7cde75ee59eec92dfa3

**Pull request merged**